tracing-opentelemetry/Cargo.toml

```
@@ -6,8 +6,8 @@ authors = [
     "Tokio Contributors <team@tokio.rs>"
 ]
 description = "OpenTelemetry integration for tracing"
-homepage = "https://github.com/tokio-rs/tracing/tracing-opentelemetry"
-repository = "https://github.com/tokio-rs/tracing/tracing-opentelemetry"
+homepage = "https://github.com/tokio-rs/tracing/tree/master/tracing-opentelemetry"
+repository = "https://github.com/tokio-rs/tracing/tree/master/tracing-opentelemetry"
```

My understanding is that the `repository` field is supposed to be _just_ the path to the repository itself, not to a subdirectory (although after looking at the [docs](https://doc.rust-lang.org/cargo/reference/manifest.html#the-repository-field) it doesn't actually say that, so I don't remember where I read this). The rest of the `tracing` crates just point to
```suggestion
repository = "https://github.com/tokio-rs/tracing"
```

---

Yeah better to match the other crates